backend/dataall/cdkproxy/stacks/environment.py

```diff
@@ -364,42 +358,96 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
             runtime=_lambda.Runtime.PYTHON_3_9,
         )
 
+        glue_db_provider = cr.Provider(
+            self,
+            f'{self._environment.resourcePrefix}GlueDbCustomResourceProvider',
+            on_event_handler=gluedb_custom_resource
+        )
+
         ssm.StringParameter(
             self,
             'GlueCustomResourceFunctionArn',
             string_value=gluedb_custom_resource.function_arn,
-            parameter_name=f'/dataall/{self._environment.environmentUri}/cfn/custom-resources/lambda/arn',
+            parameter_name=f'/dataall/{self._environment.environmentUri}/cfn/custom-resources/gluehandler/lambda/arn',
```

Comment:
Very minor - are these SSM parameters (`GlueCustomResourceFunctionName`, `GlueCustomResourceFunctionArn`) needed? Are they used?

Reply:
I had the same thought, they are not needed in any piece of code. But I have used them in the past to troubleshoot and find the original Lambda of the custom resource. I have left them on purpose but I do not have a strong opinion about them